src/webhook/handlers/utils.js

```diff
@@ -563,4 +563,40 @@ function createHighlightContent(text) {
   return result;
 }
 
+/**
+ * @param {*} GetReply
+ * @param {*} GetArticle
```

Comment:
Nit: suggest using `reply` and `article` as argument names, and denote that their type is `Reply` and `Article` in rumors-api GraphQL, respectively.